docs/resources/idp_open_id_connect.md

```diff
@@ -43,24 +49,27 @@ resource "fusionauth_idp_open_id_connect" "OpenID" {
     - `oauth2_client_secret` - (Optional) This is an optional Application specific override for the top level client secret.
     - `create_registration` - (Optional) Determines if a UserRegistration is created for the User automatically or not. If a user doesn’t exist in FusionAuth and logs in through an identity provider, this boolean controls whether or not FusionAuth creates a registration for the User in the Application they are logging into.
     - `enabled` - (Optional) Determines if this identity provider is enabled for the Application specified by the applicationId key.
-    - `oauth2_scope` - (Optional) This is an optional Application specific override for for the top level scope.
+    - `oauth2_scope` - (Optional) This is an optional Application specific override for the top level scope.
 * `button_image_url` - (Optional) The top-level button image (URL) to use on the FusionAuth login page for this Identity Provider.
 * `button_text` - (Required) The top-level button text to use on the FusionAuth login page for this Identity Provider.
 * `debug` - (Optional) Determines if debug is enabled for this provider. When enabled, each time this provider is invoked to reconcile a login an Event Log will be created.
 * `domains` - (Optional) This is an optional list of domains that this OpenID Connect provider should be used for. This converts the FusionAuth login form to a domain-based login form. This type of form first asks the user for their email. FusionAuth then uses their email to determine if an OpenID Connect identity provider should be used. If an OpenID Connect provider should be used, the browser is redirected to the authorization endpoint of that identity provider. Otherwise, the password field is revealed on the form so that the user can login using FusionAuth.
 * `enabled` - (Optional) Determines if this provider is enabled. If it is false then it will be disabled globally.
 * `lambda_reconcile_id` - (Optional) The unique Id of the lambda to used during the user reconcile process to map custom claims from the external identity provider to the FusionAuth user.
+* `linking_strategy` - (Optional) The linking strategy to use when creating the link between the {idp_display_name} Identity Provider and the user.
```

Comment:
We could optionally document the default when not specified which is `LinkByEmail` in the API doc, or whatever that maps to in go. 


https://fusionauth.io/docs/v1/tech/apis/identity-providers/openid-connect#create-an-openid-connect-identity-provider

Reply:
Place to do it would be here:

https://github.com/gpsinsight/terraform-provider-fusionauth/blob/main/fusionauth/resource_fusionauth_idp_open_id_connect.go#L131-L145

where you would end up with:
```go
			"linking_strategy": {
				Type:     schema.TypeString,
				Optional: true,
				Computed: true,
				Default: "LinkByEmail",
				...
```